base/lib/stellar/sep10.rb

```diff
@@ -62,10 +73,11 @@ def self.build_challenge_tx(server:, client:, anchor_name:, timeout: 300)
     # = Example
     #
     #   sep10 = Stellar::SEP10
-    #   challenge = sep10.build_challenge_tx(server: server, client: user, anchor_name: anchor, timeout: timeout)
-    #   envelope, client_address = sep10.read_challenge_tx(challenge: challenge, server: server)
+    #   server = Stellar::KeyPair.random # this should be the SIGNING_KEY from your stellar.toml
+    #   challenge = sep10.build_challenge_tx(server: server, client: user, home_domain: domain, timeout: timeout)
+    #   envelope, client_address = sep10.read_challenge_tx(server: server, challenge: challenge)
     #
-    def self.read_challenge_tx(challenge_xdr:, server:)
+    def self.read_challenge_tx(server:, challenge_xdr:)
```

Comment:
Did you intentionally change the order of the parameters? Why?

Reply:
Or does it not matter in ruby because they are keyword args?

Reply:
The order doesn't matter for keyword args, and I prefer maintaining consistent args structure for related methods.